protocol/0082-ETHD-ethereum-data-source.md

```diff
@@ -111,80 +111,81 @@ Select {
+9. One oracle data event for multiple matching spec id's - Create 2 markets with ethereum oracle settlement specs that use the same settlement key such that - the first settlement spec expects settlement data to be greater than 100 and the second expects greater than 200. Now send it a settlement data of 300. One single event BUS_EVENT_TYPE_ORACLE_DATA for the settlement data is emitted with both matching ethereum oracle data sources listed within the event. Both markets are settled and both the data sources are DEACTIVATED.
+10. Different oracle data events for multiple spec id's with non matching filter values - Create 2 markets with ethereum oracle settlement specs that use the same settlement key such that - the first settlement spec expects settlement data to be greater than 100 and the second expects greater than 200. Now send it a settlement data of 50. NO data events for BUS_EVENT_TYPE_ORACLE_DATA. Send settlement data of 150. One single event BUS_EVENT_TYPE_ORACLE_DATA emitted for the settlement data is emitted with matching ethereum oracle data spec for Market1, market1 is settled and the data source is set to DEACTIVATED. Send settlement data of 250. One single event BUS_EVENT_TYPE_ORACLE_DATA emitted for the settlement data is emitted with matching ethereum oracle data spec for Market2, Market2 is settled and the data source is set to DEACTIVATED.
 
 ### New Network parameters
 
-1. Test min / max values / validations for any new network parameters that are added (<a name="0082-ETHD-017" href="#0082-ETHD-017">0082-ETHD-017</a>)
-2. Test the successful disabling / enabling of ethereum oracles when the new network parameter "ethereum.oracles.enabled" is set to false or true respectively (<a name="0082-ETHD-018" href="#0082-ETHD-018">0082-ETHD-018</a>)
+1. New network parameter - ethereum.oracles.enabled. Setting this to 0 should NOT allow market creation and market updates with ethereum oracles. (<a name="0082-ETHD-017" href="#0082-ETHD-017">0082-ETHD-017</a>)
+2. New network parameter - ethereum.oracles.enabled. Setting this to 1 should allow market creation amd market updates with ethereum oracles. (<a name="0082-ETHD-018" href="#0082-ETHD-018">0082-ETHD-018</a>)
 
 ### Negative Tests
 
 1. Set up a new data source with invalid contract address - should fail validations (Phase 2 - listening for invalid event ) (<a name="0082-ETHD-019" href="#0082-ETHD-019">0082-ETHD-019</a>)
```

Comment:
is this phase 1 or phase 2?

If both, maybe should we split.

If phase 2 then maybe consider this [comment](https://github.com/vegaprotocol/specs/pull/1875/files#r1284248380)

Reply:
This is Phase 1